cmake/cpp_bindgen.cmake.in

```diff
@@ -110,7 +110,6 @@ function(cpp_bindgen_add_library target_name)
 
     add_library(${target_name} ${ARG_SOURCES})
     target_link_libraries(${target_name} PRIVATE cpp_bindgen_generator)
-    # target_include_directories(${target_name} PRIVATE ${PROJECT_SOURCE_DIR}/include) #TODO probably wrong
```

Comment:
I agree, we should remove this (it should already be part of `cpp_bindgen_generator` anyway I think.